sample-app/src/main/java/com/joinforage/android/example/ui/complete/flow/tokens/TokensViewModel.kt

```diff
@@ -2,14 +2,13 @@ package com.joinforage.android.example.ui.complete.flow.tokens
 
 import androidx.lifecycle.MutableLiveData
 import com.joinforage.android.example.ui.base.BaseViewModel
-import com.joinforage.android.example.ui.complete.flow.tokens.model.TokensUIDefaultState
 import dagger.hilt.android.lifecycle.HiltViewModel
 import javax.inject.Inject
 
 @HiltViewModel
 class TokensViewModel @Inject constructor() : BaseViewModel() {
-    val token = MutableLiveData(TokensUIDefaultState().bearer)
-    val merchantAccount = MutableLiveData(TokensUIDefaultState().merchantAccount)
+    val token = MutableLiveData("dev_1wtife770rjDwMUUbhe6jMZeEDVDtc")
```

Comment:
I think we don't want to put in a real token in the sample app

Reply:
great catch! I'll replace with a placeholder instead!

Reply:
Resolved in [03ebcd6](https://github.com/teamforage/forage-android-sdk/pull/114/commits/03ebcd676695ab408bd52f0f8baa89305a9da750)